config/initializers/us_timezones.rb

```diff
@@ -0,0 +1,7 @@
+US_TIMEZONES = [
```

Comment:
moved timezones into an initializer to make them easier to re-use (compared to being in ApplicationHelper)

and, this returns an array of arrays, with the timezone itself, plus it's `name`. this is ultimately to fix a bug when the timezone string was used with `formatted_time`.  the `name` is what is expected.

there's a test below for the regression, which i'll comment on too